features/ui/hero-section/heroSection.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+
+import styled from "styled-components";
+import { color, breakpoint } from "@styles/theme";
+import { SectionHero, BackgroundTheme } from "@typings/landingpage.types";
+
+const Hero = styled.section<{ bgTheme: BackgroundTheme }>`
+  background: ${({ bgTheme }) => {
+    switch (bgTheme) {
+      case "light-gray":
+        return color("gray", 50);
+      default:
+        return "#fff";
+    }
+  }};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 96px 0;
+  & h1 {
+    color: ${color("gray", 900)};
+    font-size: clamp(36px, 29px + 2.15vw, 60px);
+    letter-spacing: -0.02em;
+    margin: 0;
+    text-align: center;
+  }
+  & p {
```

Comment:
I'd try not to target children like this. It's usually better to create a separate styled-component. For example, if you decided to use a `div` instead of a `p` here the styles would break.